pkgs/racket-doc/scribblings/reference/syntax-model.scrbl

```diff
@@ -520,16 +520,16 @@ core syntactic forms are encountered:
  @item{When a @racket[define], @racket[define-values],
        @racket[define-syntax], or @racket[define-syntaxes] form is
        encountered at the top level or module level, a binding is
-       added @tech{phase level} 0 (i.e., the @tech{base environment}
+       added to @tech{phase level} 0 (i.e., the @tech{base environment}
```

Comment:
I think the original one makes more sense to me.

Reply:
IMO, this is a question of grammar not understanding.

Reply:
Ah, my bad! I missed the word "is" and read the sentence as "a binding added to phase level", which didn't make sense to me. But with "is", adding "to"/"into" is definitely the right thing to do.